### PR TITLE
Add Bahasa Melayu (ms) support

### PR DESCRIPTION
**Pull request by saga0324**

### CODE REVIEW

File: app/locales/index.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import cs from "./cs";
 import ko from "./ko";
 import ar from "./ar";
 import bn from "./bn";
+import new from "./new.ts";
```
Comment (H0llyW00dzZ) [inline app/locales/index.ts:19]: change this to  `my`

File: app/locales/index.ts [hunk 2]
```diff
@@ -40,6 +41,7 @@ const ALL_LANGS = {
   no,
   ar,
   bn,
+  new
```
Comment (H0llyW00dzZ) [inline app/locales/index.ts:44]: this as well changed to `my`

File: app/locales/index.ts [hunk 3]
```diff
@@ -65,6 +67,7 @@ export const ALL_LANG_OPTIONS: Record<Lang, string> = {
   no: "Nynorsk",
   ar: "العربية",
   bn: "বাংলা",
+  new: "new lang",
```
Comment (H0llyW00dzZ) [inline app/locales/index.ts:70]: this as well changed to `my`

File: app/locales/new.ts [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+      Topic: "Topik",
+      Time: "Tarikh & Waktu",
+    },
+    URLCommand: {
+      Code: "Kod akses terdeteksi dari url, sahkan untuk mendaftar? ",
+      Settings: "Tetapan terdeteksi dari url, sahkan untuk diterapkan?",
+    },
+  };
+  
+  export default new;
```
Comment (H0llyW00dzZ) [inline app/locales/new.ts:385]: this as well changed to `my`

**Review by H0llyW00dzZ — COMMENTED** [review]
need change `new` to `my`

**Comment by H0llyW00dzZ** [comment]
> > need change `new` to `my`
> 
> Thanks a lot. But I think the ISO 639-1 code of Bahasa Melayu is MS, not MY

it's fine because `my` used to be code name of country
`my` = malaysia

**Comment by H0llyW00dzZ** [comment]
nvm yeah you can changed to ms instead of my

**Review by saga0324 — COMMENTED** [review]
Fix MY to MS

**Review by H0llyW00dzZ — COMMENTED** [review]
great

**Review by H0llyW00dzZ — CHANGES_REQUESTED** [review]
this should be `ms` not a `new`

**Review by saga0324 — COMMENTED** [review]
Fixed

**Review by H0llyW00dzZ — APPROVED** [review]
nice

**Pull request closed without merging**